templates/js/custom-copilot-assistant-assistants-api/appPackage/manifest.json.tpl

```diff
@@ -29,7 +29,24 @@
                 "personal"
             ],
             "supportsFiles": false,
-            "isNotificationOnly": false
+            "isNotificationOnly": false,
+            "commandLists": [
+                {
+                    "scopes": [
+                        "personal"
+                    ],
+                    "commands": [
+                        {
+                            "title": "Solve the equation: 3x + 11= 14",
```

Comment:
All commands are tested and workable?

Reply:
all tested, only “What's weather in redmond, us?” bot can't get the answer
<img width="867" alt="image" src="https://github.com/user-attachments/assets/15b2a971-f91a-44ce-861b-78abbc8a2fa9">

Reply:
Update to a workable prompt.

Reply:
"Tell me the weather of San Francisco" is a possible workable prompt.

Reply:
ok, but as the length limit, update command to "The weather of San Francisco"